client/factory.go

```diff
@@ -91,5 +91,10 @@ func NewBucket(logger log.Logger, confContentYaml []byte, reg prometheus.Registe
 		return nil, errors.Wrap(err, fmt.Sprintf("create %s client", bucketConf.Type))
 	}
 
-	return objstore.NewTracingBucket(objstore.BucketWithMetrics(bucket.Name(), objstore.NewPrefixedBucket(bucket, bucketConf.Prefix), reg)), nil
+	return objstore.NewPrefixedBucket(bucket, bucketConf.Prefix), nil
+}
+
+// NewInstrumentedBucket initializes and returns new object storage clients.
```

Comment:
This is not true - it only wraps with metrics. Should we rather remove `NewInstrumentedBucket` and keep `objstore.BucketWithMetrics`?